ui/components/app/edit-gas-fee-popover/network-status/network-status.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { toBigNumber } from '../../../../../shared/modules/conversion.utils';
+import { COLORS } from '../../../../helpers/constants/design-system';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import I18nValue from '../../../ui/i18n-value';
+import Typography from '../../../ui/typography/typography';
+
+import StatusSlider from './status-slider';
+
+const NetworkStatus = () => {
+  const { gasFeeEstimates } = useGasFeeContext();
+  let estBaseFee = null;
+  if (gasFeeEstimates?.estimatedBaseFee) {
+    // estimatedBaseFee is not likely to be below 1, value .01 is used as test networks sometimes
+    // show have small values for it and more decimal places may cause UI to look broken.
+    estBaseFee = toBigNumber.dec(gasFeeEstimates?.estimatedBaseFee);
+    estBaseFee =
+      estBaseFee.greaterThan(0.01) < 0.01 ? 0.01 : estBaseFee.toFixed(2);
```

Comment:
`greaterThan`/`lessThan` already return a boolean, so you don't need to combine it with `<` or `>`. But also, should this be `estBaseFee.lessThan(0.01) ? estBaseFee : estBaseFee.toFixed(2)`? So basically, the thinking here is that if it's a very small value we go ahead and show that value, otherwise we chop it off at 2 decimal places.

Reply:
yep that was my bad, just fixed it. thx